src/pages/landing/supporters/index.jsx

```diff
@@ -55,24 +55,11 @@ const Supporters = () => {
             />
           </a>
           <p className="landing-section-paragraph">
-            <a

```

Comment:
why did you remove the links for FIU and UPE?

Reply:
Because they looked ugly and were unnecessary